### ISSUE TITLE
Accessibility Contrast Issues

### ISSUE THREAD

**Issue by Dipanita45** [Tushar-sonawane06/UI-Verse#444]
Some light gray text and pastel tag colors may fail accessibility contrast standards in dark mode.
